resource_reaper.py

```diff
@@ -0,0 +1,147 @@
+'''
+Automated Vertex AI resource cleanup for python-docs-samples-test
+'''
+
+# ! pip3 install google-cloud-aiplatform google-cloud-bigquery --quiet
+
+from google.cloud import aiplatform
+from google.cloud import bigquery
+import time
+import datetime
+
+PROJECT_ID = "python-docs-samples-tests"  # @param {type:"string"}
```

Comment:
question: typically we pull the project ID from the environment variables. Can you do that here?

Reply:
suggestion: I think you can also delete the param type hint, since this isn't to be used in Colab.